src/components/modal/modalHeader/modalHeader.tsx

```diff
@@ -7,15 +7,28 @@ const cx = classNames.bind(styles);
 
 interface ModalHeaderProps {
   title?: ReactNode;
+  headerContentClassName?: string;
   headerNode?: ReactNode;
+  headerDescription?: string;
   onClose: () => void;
 }
 
-export const ModalHeader: FC<ModalHeaderProps> = ({ title, onClose, headerNode }) => (
+export const ModalHeader: FC<ModalHeaderProps> = ({
+  title,
+  headerContentClassName = '',
+  headerDescription,
+  onClose,
+  headerNode,
+}) => (
   <div className={cx('modal-header')}>
-    <div className={cx('modal-header-content')}>
+    <div className={cx('modal-header-content', headerContentClassName)}>
       {title && <span className={cx('modal-title')}>{title}</span>}
       {headerNode && headerNode}
+      {headerDescription ? (
+        <span className={cx('modal-header-description')}>{headerDescription}</span>
+      ) : (
+        <div className={cx('modal-header-spacer')} />
+      )}
```

Comment:
1. I suppose the description can have some style customizations (see the screenshot below), so may be make sense to make it as a part of the content?
![image](https://github.com/user-attachments/assets/d2cd975c-d50c-4d64-b3e9-280ca3af1cb3)
In this case it will also not break the scroll area calculation as we assume that the header height is fixed, but content is flexible.
2. It is redundant to create an empty html element like `modal-header-spacer` when we can make indentation with paddings/margins.